src/core/server/opensearch/version_check/ensure_opensearch_version.test.ts

```diff
@@ -244,7 +247,12 @@ describe('pollOpenSearchNodesVersion', () => {
       });
   });
 
-  it('starts polling immediately and then every opensearchVersionCheckInterval', () => {
+  /*
+   * Disable TestScheduler test due to OOM issue,
+   * refer to https://rxjs.dev/guide/testing/marble-testing#known-issues
+   */
+  // TODO: investigate removing or rewriting this unit test
```

Comment:
In spirit of what was suggested in a previous PR, can we create an issue to return back to fixing these tests?

Reply:
Yes, will create the issues